server/src/database/database.js

```diff
@@ -1,17 +1,25 @@
-import mongoose from 'mongoose'; 
+import mongoose from 'mongoose';
 import config from '../config/environment';
 import message from '../commons/constants/messages';
 
+/**
+ * Class to set the connection with the Data Base.
+ */
 export default class DataBaseConnection {
 
     constructor() {
+        this.message = message;
         this.URI = `${config.uri}${config.db}`;
     }
-    
+
+    /**
+     * Get the connection.
+     * @returns{mongoose} the connection
+     */
     getDataBaseConnection() {
         mongoose.connect(this.URI)
-            .then(db => console.log(message.dbConnected))
-            .catch(err => console.log(err));
+        .then(() => console.info(this.message.dbConnected))
```

Comment:
sorry now I see `message` is not a parameter of the arrow fucntion, but import constant, please revert this change.